USAGE.md

```diff
@@ -0,0 +1,104 @@
+# Amazon Alexa Smart Home Skill for openHAB 2
+
+Amazon certified openHAB2 as a [smarthome skill for Alexa](https://www.amazon.com/openHAB-Foundation/dp/B01MTY7Z5L).
+This page describes how to use the skill certified by Amazon.
+This skill allows you to connect your openHAB setup through the [myopenHAB.org](http://myopenHAB.org) cloud service to Amazon Alexa.
+(See [README.md](https://github.com/openhab/openhab-alexa/blob/master/README.md) for other setup options regarding your own openHAB cloud.)
```

Comment:
openHAB cloud is not required if you host it your self, i think it would be better to say:

"See [README.md](https://github.com/openhab/openhab-alexa/blob/master/README.md) for other setup options and development information.)"